rosdep/python.yaml

```diff
@@ -6051,6 +6042,13 @@ python3-rosdistro-modules:
   openembedded: [python3-rosdistro@meta-ros]
   rhel: ['python%{python3_pkgversion}-rosdistro']
   ubuntu: [python3-rosdistro-modules]
+python3-rosinstall-generator:
+  debian:
+    '*': [python3-rosinstall-generator]
+    stretch: null
+  ubuntu:
+    '*': [python3-rosinstall-generator]
+    xenial: null
```

Comment:
Is there a reason this key is nulled for Xenial and Stretch? I'm able to install the python3 package for rosinstall_generator on both platforms?

Reply:
Turned out the nulled entries were the suggestion (that I accepted without reviewing. My bad)  in https://github.com/ros/rosdistro/pull/25482#discussion_r441018846.  @clalancette care for the rationale?

Reply:
As far as I can tell, it isn't available in Xenial: https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=python3-rosinstall-generator&searchon=names

Nor in Stretch: https://packages.debian.org/search?keywords=python3-rosinstall-generator&searchon=names&suite=all&section=all

Reply:
Those packages are not in the upstream, but we provide instances of them in our repositories. For example: http://packages.ros.org/ros/ubuntu/dists/xenial/main/binary-amd64/Packages

Thus the keys are considered valid for rosdeps.